Python/bytecodes.c

```diff
@@ -3113,6 +3185,7 @@ dummy_func(
 
         inst(MAKE_FUNCTION, (defaults    if (oparg & 0x01),
                              kwdefaults  if (oparg & 0x02),
+                             typevars    if (oparg & 0x10),
```

Comment:
We need fewer flags, not more.

I think the best approach would be to set attributes on the function after it is created.
That way inner functions can be created quickly and we only pay for the extra bits we need.

Reply:
Thanks, I can switch to that approach here.

Reply:
Done